test/e2e/flowaggregator_test.go

```diff
@@ -120,6 +121,8 @@ const (
 	testIngressRuleName            = "test-ingress-rule-name"
 	testEgressRuleName             = "test-egress-rule-name"
 	iperfTimeSec                   = 12
+	iperfTimeSecShort              = 5
```

Comment:
I saw this number gets modified two or three times, is there a special reason behind?

Reply:
At the beginning I planned to check the `PodLabels` field in the flow record from `antctl`. But the field is only added when the record is going to be sent, and the `antctl` can only get the flow records still stored in the `AggregationProcess`.  Thus I was looking for a shortest time period which can ensure the field is added while the record is not deleted from the `AggregationProcess`.  I have tried some time period which works locally, but they might not work on CI. 

After some discussions with Yongming, I decided not to check the `PodLabels` field. This allows us to use a shorter `iperfTime` and provides a more stable test result. 5 is selected as I think it is a time that the first record is not sent but is ready in the `AggregationProcess`.

Reply:
Thanks for the explanation. My recommendation is to use a *local* constant in the test case that requires this value and add a comment there explaining why this value was chosen. IMO, having a global constant like this one can hurt the readability of the tests and doesn't provide much value.

Reply:
Updated